sphinx/ext/autodoc/__init__.py

```diff
@@ -840,8 +859,8 @@ def parse_name(self) -> bool:
                            type='autodoc')
         return ret
 
-    def add_directive_header(self, sig: str) -> None:
-        Documenter.add_directive_header(self, sig)
+    def add_directive_header_multi(self, sigs: List[str]) -> None:
+        Documenter.add_directive_header(self, sigs)
```

Comment:
Note: It would be better to use `super()` instead after this PR.

Reply:
Done in this PR.